kubejobs/__init__.py

```diff
@@ -195,8 +195,10 @@ def update_env_vars(self, data):
         data['env_vars']['REDIS_HOST'] = 'redis-%s' % self.app_id
 
         # inject SCONE_CONFIG_ID in the environment
-        # FIXME: make SCONE_CONFIG_ID optional in submission
-        data['env_vars']['SCONE_CONFIG_ID'] = data['config_id']
+        config_id = data.get('config_id')
+        if (config_id):
```

Comment:
```suggestion
        if config_id is not None:
```
https://docs.quantifiedcode.com/python-anti-patterns/readability/test_for_object_identity_should_be_is_not.html?highlight=none